dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -24,5 +24,69 @@
 #   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
 #   (single quote must be coded as one single quote '). 
 
-# All messages must use the range CWWKD1100 to CWWKD1499
+# All messages must use the range CWWKD1000 to CWWKD1399
+
+CWWKD1000.repo.general.err=CWWKD1000E: The {0} method of the {1} repository \
+ interface encountered an error indicating that the repository method might \
+ not be valid. The error is: {2}.
+CWWKD1000.repo.general.err.explanation=An error was found with the specified \
+ repository method.
+CWWKD1000.repo.general.err.useraction=Locate the repository interface and method \
+ and determine whether corrections are needed.
+
+CWWKD1001.no.primary.entity=CWWKD1001E: An entity class is needed by the {0} \
+ repository method, but the {1} repository interface does not specify a primary \
+ entity class. To correct this, have the {1} interface extend one of the \
+ built-in repository supertypes, such as {2}, supplying the entity class as \
+ the first type parameter.
+CWWKD1001.no.primary.entity.explanation=Repository methods without an entity \
+ class of their own require a primary entity class to be defined by the \
+ repository.
+CWWKD1001.no.primary.entity.useraction=Define a primary entity class for the \
+ repository.
+
+CWWKD1002.method.annos.err=CWWKD1002E: The {0} method of the {1} repository \
+ interface is annotated with a combination of annotations, {2}, that is not \
```

Comment:
```suggestion
 interface is annotated with a combination of annotations, {2}, that are not \
```

The subject `combination of annotations` is a singular idea, but in spoken English `is not` sounds wrong because the subject is still referring to multiples.  I'm guessing it could go either way maybe the ID team can chime in on this.

Reply:
I'll leave this one to the ID team. If we have a choice, I prefer "is" which refers to the combination is not supported, rather than "are", which makes it sound like the annotations are not supported.  The former is accurate, whereas the latter is not.  The annotations themselves might all be supported, bu the combination of them isn't.

Reply:
I agree with nathan here, if we are saying this _combination_ of annotations is invalid (singular collective noun) as opposed to all the annotations each being invalid on their own (plural group). Either way, I wouldn't split the clause with the list of annotations.

The {0} method of the {1} repository interface is annotated with a combination of annotations that is not valid: {2}
or
The {0} method of the {1} repository interface is annotated with an invalid combination of annotations: {2}

Reply:
Updating as suggested: The {0} method of the {1} repository interface is annotated with a combination of annotations that is not valid: {2}